src/main/java/com/minecolonies/coremod/client/gui/WindowWorkerMultiPages.java

```diff
@@ -0,0 +1,74 @@
+package com.minecolonies.coremod.client.gui;
```

Comment:
Beware of the almost same existing mechanic for global page system. Might have name (or id) collisions in xmls and methods

Reply:
<https://github.com/ldtteam/minecolonies/blob/e3016c19e2ce1ed310979ef164f3eaa633f79aa4/src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowSkeleton.java>

Reply:
From my sight, this class is unnecessary since handled by super

Reply:
Didn't know this was there already =O

Reply:
How would huts pages work then? :D

Reply:
Now I can comment again, I removed the manual switching from a few huts then =D